test/unit/mods.example.ts

```diff
@@ -264,4 +264,92 @@ export const ArchiveOnly = new Map<string, ExampleMod>(
   }), // object
 );
 
-export const AllModTypes = [CetMod, ArchiveOnly];
+export const jsonMod = new Map<string, ExampleMod>(
+  Object.entries({
+    jsonWithValidFileInRoot: {
+      inFiles: ["giweights.json"].map(path.normalize),
+      outInstructions: [
+        {
+          type: "copy",
+          source: path.normalize("giweights.json"),
+          destination: path.normalize("engine/config/giweights.json"),
+        },
+      ],
+    },
+    jsonWithInvalidFileInRoot: {
```

Comment:
Rather than invalid, is this more of an "unknown" file since it gets moved anyway? Same above, I guess, and also relating to how you want to conceptualize the "known" files for JSON

Reply:
It's invalid because we have no way of knowing where to put it short of it already being where it belongs

Reply:
I should add that that test passes when it shouldn't but testing actual files in Vortex has my expected behavior (rejecting).  Maybe you can see what I did wrong.